focal/4_gs1_identification.html

```diff
@@ -0,0 +1,336 @@
+            the length up to 12 digits), and a check digit (a mathematical calculation to detect keying errors).
+      </p>
+      <p>
+            The GS1 Company Prefix is licensed to a user company by a local GS1 Member Organization. The license gives
+            the user company the right to issue GS1 identification keys within the range of the GS1 Company Prefix and
+            to issue Verifiable Credentials referring to the license Verifiable Credential.
+      </p>
+      <p>
+            The license Verifiable Credential may be revoked if the user company fails to abide by the terms and
+            conditions or may be transferred to another user company as part of a merger and acquisition.
+      </p>
+      <p>
+            If the license is revoked, no new Verifiable Credentials derived from it may be issued. If the license is
+            transferred, existing derived Verifiable Credentials remain valid, and the new user company may issue new
+            Verifiable Credentials, including some that refer to Verifiable Credentials issued by the previous user
+            company.
+      </p>
+      <div class="note">
+            <p>
+                  The core GS1 standard is the identification of objects in the supply chain, typically trade items, but also
+                  locations, shipping containers, and much more. Every object is identified using a GS1 identification key,
+                  sometimes alongside a secondary key for higher granularity (e.g., a serial number alongside a GTIN to
+                  identify a specific instance of a trade item). Much of the text in this use case refers to keys and key
+                  credentials. These are the GS1 identification keys, <strong>not</strong> cryptographic keys.
+            </p>
+      </div>
+      <h4>Distinction</h4>
+      <p>
+            This differs from other focal use cases in that the rights granted by a Verifiable Credential can be
+            transferred
+            to another subject, without invalidating other Verifiable Credentials created by the original subject.
+      </p>
+      <h4>Scenario</h4>
+      <p>
+            Healthy Tots, a baby food manufacturer, wishes to list its products on the Sell Anything &amp; Everything
+            (SA&amp;E) marketplace. As a global marketplace, SA&amp;E requires unique identification for products listed
+            on its site, and has chosen the GTIN as the preferred identification key. To ensure uniqueness, SA&amp;E
+            requires that companies listing products prove that they have the right to issue the GTINs they are using.
+      </p>
+      <h4>Verifiable Credentials</h4>
+      <dl class="dl-horizontal">
+            <dt>
+                  GS1 Prefix license
+            </dt>
+            <dd>
+                  Issued by GS1 Global Office to GS1 Utopia (a GS1 Member Organization). Grants GS1 Utopia the right to
```

Comment:
```suggestion
                  Issued by GS1 Global Office to GS1 Utopia (a GS1 Member Organization operating in the State of Utopia). Grants GS1 Utopia the right to
```

I like "Utopia" as an arbitrary state, whether nation-state or province. This is explained in the actors section below, but a few words here would have helped the flow when I first read this.